lib/clean.js

```diff
@@ -1,4 +1,8 @@
 const argParser = require('yargs')
+
+const genCore = require('./gen-core')
 const cli = require('./cli')
+
 const argv = argParser(process.argv.slice(2)).argv
+genCore()
```

Comment:
@angadgill92 Do you think it is useful? After rebase this would not be required for the reason this is done.

In case you recommend doing this, now I minified using both `esprima`, and `escodegen`. Please suggest a good, simple minifyer.